espnet2/bin/k2_asr_inference.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python3
```

Comment:
I forgot to mention it in the previous PR.
I think it’s better to rename it `asr_inference_k2.py`.
Same prefix of the function name makes extensions easier.

E.g., If we have other inference engine (e.g., k3), we may add asr_inference_k3.py, and people can easily understand that k3 is a variant of the decoding method.
If we extend k2 to add some new functions, e.g., say k2_fast, then asr_inference_k2_fast.py.